src/SprykerEco/Yves/Unzer/Plugin/StepEngine/UnzerMarketplaceSofortSubFormPlugin.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Yves\Unzer\Plugin\StepEngine;
+
+use Spryker\Yves\Kernel\AbstractPlugin;
+use Spryker\Yves\StepEngine\Dependency\Form\StepEngineFormDataProviderInterface;
+use Spryker\Yves\StepEngine\Dependency\Form\SubFormInterface;
+use Spryker\Yves\StepEngine\Dependency\Plugin\Form\SubFormPluginInterface;
+
+/**
+ * @method \SprykerEco\Yves\Unzer\UnzerFactory getFactory()
+ */
+class UnzerMarketplaceSofortSubFormPlugin extends AbstractPlugin implements SubFormPluginInterface
+{
+    /**
+     * {@inheritDoc}
+     * - Creates `Marketplace BankTransfer` subform.
+     *
+     * @api
+     *
+     * @return \Spryker\Yves\StepEngine\Dependency\Form\SubFormInterface
+     */
+    public function createSubForm(): SubFormInterface
+    {
+        return $this->getFactory()->createMarketplaceSofortSubForm();
+    }
+
+    /**
+     * {@inheritDoc}
+     * - Creates `Marketplace BankTransfer` subform data provider.
```

Comment:
Please, fix the description - Creates `Marketplace Sofort` subform data provider.

Reply:
Fixed.